### ISSUE TITLE
No scroll on < 787 px width view

### ISSUE THREAD

**Issue by barneycarroll** [fat/coffin#1]
No mousewheel scroll on Chrome, IE, FF — arrow key scroll only in FF. Tested on github demo page.

**Comment by matthewrobertson** [comment]
confirmed on Chrome and Chrome Canary on OSX

**Comment by fat** [comment]
yeah, i think scrolling should be done entirely with translate3d. You shouldn't rely on native scroll– it's too bad

**Comment by dbaq** [comment]
confirmed on Chrome and Safari. No updates?
